CatAnalyzer/plugins/ttbbLepJetsAnalyzer.cc

```diff
@@ -677,7 +1039,8 @@ void ttbbLepJetsAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetu
   AnalysisHelper trigHelper = AnalysisHelper(triggerNames, triggerBits, triggerObjects);
 
   if ( (ch_tag == 0 && (trigHelper.triggerFired("HLT_IsoMu20_v") || trigHelper.triggerFired("HLT_IsoTkMu20_v"))) ||
-       (ch_tag == 1 && (trigHelper.triggerFired("HLT_Ele22_eta2p1_WP75_Gsf_v") || trigHelper.triggerFired("HLT_Ele22_eta2p1_WPLoose_Gsf_v"))) ) {
+       (ch_tag == 1 ) ) { // TO BE UPDATED: Add electron triggers
+    //(ch_tag == 1 && (trigHelper.triggerFired("HLT_Ele22_eta2p1_WP75_Gsf_v") || trigHelper.triggerFired("HLT_Ele22_eta2p1_WPLoose_Gsf_v"))) ) {
     EvTrigger = true;
   }
```

Comment:
By the way, HLT_IsoMu20 does not exist anylonger for Run2016D and E. 
We can increase the threshold IsoMu27 even for 2016B and 2016C?